docs/content/en/docs/user-guide/configuration-reference.md

```diff
@@ -198,6 +203,19 @@ One of `yamlField` or `regex` is required.
 | timeout | duration | The maximum time the stage can be taken to run. | No |
 | with | [StageOptions](/docs/user-guide/configuration-reference/#stageoptions) | Specific configuration for the stage. This must be one of these [StageOptions](/docs/user-guide/configuration-reference/#stageoptions). | No |
 
+## DeploymentNotification
+
+| Field | Type | Description | Required |
+|-|-|-|-|
+| mentions | [][NotificationMention](/docs/user-guide/configuration-reference/#notificationmention) | List of users to be notified for each event. | No |
+
+## NotificationMention
+
+| Field | Type | Description | Required |
+|-|-|-|-|
+| event | []string | The event to be notified to users. | Yes |
+| slack | []string | The user name for mentioning in Slack. | Yes |
```

Comment:
user id instead of usename if I recall correctly.

Reply:
1f75a04